### PR TITLE
add navbar

### PR DESCRIPTION
**Pull request by amiqbal7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +12,34 @@
...
+            <li class="nav-item">
+              <a class="nav-link" href="#why">Why us</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#testi">Testimonials</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#faq">FAQ</a>
+            </li>
+          </ul>
```
Comment (ardyanandakirana) [inline index.html:38]: please, add navbar item contact

**Review by ardyanandakirana — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (amiqbal7) [reply index.html:38]: okey

File: index.html [hunk 2]
```diff
@@ -12,6 +12,36 @@
     />
   </head>
   <body>
+    <nav class="navbar navbar-expand-lg bg-body-tertiary">
+      <div class="container-fluid px-3" style="background-color: #F1F3FF;">
```
Comment (CoreX20) [inline index.html]: change background color to :  #e5e5e5

**Review by CoreX20 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (amiqbal7) [reply index.html]: oke

**Pull request merged**